demo/plugins/lookup_plugins/conjur_variable.py

```diff
@@ -0,0 +1,156 @@
+#!/usr/bin/python
+
+import os.path
+import ssl
+import yaml
+from ansible.plugins.lookup import LookupBase
+from base64 import b64encode
+from httplib import HTTPConnection
+from netrc import netrc
+from os import environ
+from sys import stderr
+from time import time
+from urllib import quote_plus
+from urlparse import urlparse
+
+
+def exit_error(err):
+    stderr.write(err)
+    exit(1)
+
+
+class Token:
+    def __init__(self, http_connection, id, api_key, account):
+        self.http_connection = http_connection
+        self.id = id
+        self.api_key = api_key
+        self.token = None
+        self.refresh_time = 0
+        self.account = account
+
+    # refresh
+    # Exchanges API key for an auth token, storing it base64 encoded within the
+    # 'token' member variable. If it fails to obtain a token, the process exits.
+    def refresh(self):
+        authn_url = '/authn/%s/%s/authenticate' % (quote_plus(self.account), quote_plus(self.id))
```

Comment:
We should use the `'mystring {0}'.fmt(mydata)` when we need to do string formatting in Python.

https://docs.python.org/2/library/stdtypes.html#str.format

Simple, powerful, works in Python 2 and 3, and it's what most Pythonistas recommend.